src/components/Logo/Logo.tsx

```diff
@@ -1,32 +1,37 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
 import { useTranslation } from 'react-i18next'
-import homeLogo from '../../styles/assets/images/homeLogo.svg'
-import logo from '../../styles/assets/images/logo.svg'
-import logoMobile from '../../styles/assets/images/logo_mobile.svg'
-import useDetectMobileOrTablet from '../../hooks/useDetectMobileOrTablet'
+import defaultLogo from '../../styles/assets/images/logo.svg'
+import mobileLogo from '../../styles/assets/images/logo_mobile.svg'
+import { isAndroid, isIos } from '../../services/platform'
+import styled from 'styled-components'
+
+const Img = styled.img`
+    width: 100%;
+`
+
+const StyledLink = styled(Link)`
+    width: 100px;
+    display: block;
+    @media screen and (min-width: 1024px) {
+        width: 181px;
+        height: 66px;
+    }
+`
 
 function Logo() {
     const { t } = useTranslation()
-    const isMobile = useDetectMobileOrTablet()
-    let logoToChoose = ''
+    const isMobile = isIos() || isAndroid()
```

Comment:
I think that is not exactly the same as `useDetectMobileOrTablet`:
- `useDetectMobileOrTablet` -> uses screen width
- `isIos` and `isAndroid` is provided by `capacitor`

Reply:
An oversight. I fix it